kickstart/etc/nginx-posm.conf

```diff
@@ -24,18 +24,54 @@ server {
   }
 
   # proxied locations
+
+  location ~ ^(/osm)?/api/0.6/map$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?/api/0.6/(nodes|ways|relations)$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?/api/0.6/(way|relation)/([^/]+)/full$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?(/api/0.6/(way|relation)/([^/]+))$ {
```

Comment:
@jflemer-ndp this doesn't work with the `/osm` prefix. It's passed through FastCGI correctly, but CGImap isn't interpreting the request URI.

I tried:

```
fastcgi_param REQUEST_URI $2 # or $2?$args
```

without success. Ideas?

See also https://github.com/zerebubuth/openstreetmap-cgimap/blob/e86a47e0791143dc41c6276c3ea9679f211be13c/src/request_helpers.cpp#L59

This is mainly important when using CGImap w/ the OSM instance's iD (since that uses prefixed API urls).